modules/io/common/src/main/java/org/locationtech/jts/io/twkb/BufferedTKWBOutputStream.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Gabriel Roldan, 2022 Aurélien Mino
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.locationtech.jts.io.twkb;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+class BufferedTKWBOutputStream extends TWKBOutputStream {
```

Comment:
Are `BufferedTKWBOutputStream` and `TWKBOutputStream` getting introduced just to add the methods which let one write a VarInt? 

I'm lukewarm on that.  In my opinion, I think I'd rather see `Varint.writeUnsignedVarInt(value, out);` be able to operate on an OutputStream.  Maybe there's some reason that's not easy...

Reply:
Great suggestion. I've removed these classes in https://github.com/locationtech/jts/pull/854/commits/510299dab603e1bffdfebc4d9da94477f731d25c